O3exp/pipeline/analysis.ini

```diff
@@ -72,6 +72,7 @@ do-trigger-fitting =
 ;background-bins = all:total:lt600
 
 [workflow-coincidence-full]
+timeslide-precedence = H1, L1, V1, K1, I1
```

Comment:
can you propose a comment line here to describe what this option does?

Reply:
It took more than a line, but I think this description should be clear enough, is it ok?

Reply:
@ahnitz  can I check what this actually means, without the 'pivot'/'fixed' jargon : out of any given combination of detectors, with the current implementation *only one* ifo is slid relative to all the others, and that ifo is called 'pivot' ? is that correct?

Reply:
@tdent Yes, you can think of it that way.

Reply:
OK, would there be any objection to documenting it in the ini file simply as this : 

"for a given ifo combination, time shifted coincidences are generated by shifting one ifo relative to all the rest: that ifo is chosen to be the one appearing first in this list"

The user doesn't need to know everything that goes on under the hood given that only the choice of 'pivot' makes a difference at present.  If/when the 'pivot' and 'fixed' designations mean something more specific we can update this.